geode-core/src/main/java/org/apache/geode/examples/security/ExampleSecurityManager.java

```diff
@@ -257,15 +257,65 @@ private void readUsers(final Map<String, User> rolesToUsers, final JsonNode node
     return roleMap;
   }
 
-  static class Role {
+  public static class Role {
     List<ResourcePermission> permissions = new ArrayList<>();
+
```

Comment:
They are actually needed by the test.  I can either set the fields public, move the test to the same package, or leave the accessor methods.  Which is preferred?